test/zdtm.py

```diff
@@ -895,7 +902,9 @@ def run(action,
             timeout=60):
         env = dict(
             os.environ,
-            ASAN_OPTIONS="log_path=asan.log:disable_coredump=0:detect_leaks=0")
+            ASAN_OPTIONS="log_path=asan.log:disable_coredump=0:detect_leaks=0",
+            CRIU_LIBS_DIR=PLUGINS_DIR
```

Comment:
I don't think it is a good idea to run all tests with these plugins by default... I think it should be optional.

Reply:
Thank you for the code review Andrei!

I have updated the pull request to make running ZDTM tests with plugins optional. However, I noticed that, for some reason, when running the tests only with the `amdgpu_plugin`, we still see messages from the `cuda_plugin`:

```
sudo ./zdtm.py run -t zdtm/static/busyloop00 --criu-plugin amdgpu -f h
userns is supported
=== Run 1/1 ================ zdtm/static/busyloop00
======================= Run zdtm/static/busyloop00 in h ========================
Start test
./busyloop00 --pidfile=busyloop00.pid --outfile=busyloop00.out
Run criu dump
=[log]=> dump/zdtm/static/busyloop00/56/1/dump.log
------------------------ grep Error ------------------------
b'(00.008948) cg:     `- [net_cls] -> [/] [0]'
b'(00.008950) cg: Set 1 is criu one'
b'(00.008963) Detected cgroup V1 freezer'
b"(00.008966) plugin: `cuda_plugin' hook 10 -> 0x7f0204737ede"
b'(00.052279) cuda_plugin: Error initializing CUDA: "no CUDA-capable device is detected"'
b'(00.052309) Error (cuda_plugin.c:201): cuda_plugin: Failed to launch cuda-checkpoint to retrieve restore tid: Error initializing CUDA: "no CUDA-capable device is detected"'
b'(00.052408) Collected (4 attempts, 0 in_progress)'
b'(00.052424) Collected (4 attempts, 0 in_progress)'
b'(00.052432) Collected 56 in 1 state'
b"(00.052434) plugin: `cuda_plugin' hook 11 -> 0x7f0204737cbf"
b'(00.072347) cuda_plugin: Error initializing CUDA: "no CUDA-capable device is detected"'
b'(00.072381) Error (cuda_plugin.c:201): cuda_plugin: Failed to launch cuda-checkpoint to retrieve restore tid: Error initializing CUDA: "no CUDA-capable device is detected"'
------------------------ ERROR OVER ------------------------
```

Reply:
I've updated this patch to add `--criu-plugin` option that allows to run tests with plugins.